new_coffee/coffee.py

```diff
@@ -0,0 +1,38 @@
+class Coffee:
+    """
+    Coffee class with prepared coffee quantity and time
+    """
+    def __init__(self, coffee_spoons, sugar, water):
+        self.coffee_spoons = coffee_spoons
+        self. sugar = sugar
+        self.water = water
+    quantity = 0
```

Comment:
variables declaration should be done before methods declarations,  right after the class's doc string.